src/routeApp/validations.js

```diff
@@ -0,0 +1,65 @@
+import { body } from 'express-validator';
+
+const routeValidation = () => [
+	body('origin', 'origin_required').notEmpty(),
+	body('destination', 'destination_required').notEmpty(),
+	body('distance', 'distance_required')
+		.notEmpty()
+		.isNumeric()
+		.withMessage('number')
+		.isLength({
+			max: 5,
+		})
+		.withMessage('greater_5'),
+	body('start_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('end_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('stop_points', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+];
+export const editValidation = () => [
+	body('origin', 'origin_required').optional().notEmpty(),
+	body('destination').optional().notEmpty().withMessage('destination_required'),
+	body('distance')
+		.notEmpty()
+		.withMessage('distance_required')
+		.isNumeric()
+		.withMessage('number')
+		.isLength({
+			max: 5,
+		})
+		.withMessage('greater_5'),
+	body('start_coor', 'geographical_coordinates')
+		.optional()
```

Comment:
This validation will not be possible. I will tryto find a fitting **Regular expression** in the mean time, set the min to be `10` and max to be `50` same as the **`end_coor`. This is an example of a valid geographical point

Point: **`-1.92800534178346, 30.07661466134904`**

Reply:
But I saw a geographical point with only 7 digits